agent/internal/docker.go

```diff
@@ -287,6 +290,46 @@ func (d *dockerActor) sendPullLogs(ctx *actor.Context, r io.Reader) error {
 	return scanner.Err()
 }
 
+func (d *dockerActor) addProxy(config *dcontainer.Config) {
+	validVars := map[string]string{
+		"HTTP_PROXY": "", "HTTPS_PROXY": "",
+		"FTP_PROXY": "", "NO_PROXY": "",
+	}
+
+	// if already exists, we don't modify
```

Comment:
blocking: Generally we try to have comments be proper sentences with capital letters and periods.  Lol you can ask Danny how often he has corrected my PRs on this.

Separately, this comment addresses the question of "what is this code doing".  In this case, the code is clear enough that we probably don't need to worry about that.  But the question of "why are we doing this" is important and definitely warrants a comment, to explain that we intend to give preference to the experiment config over the agent's environment.